code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -282,9 +287,11 @@
 
 	stab_direction = turn(get_dir(xeno, targeted_atom), 180)
 	playsound(hit_target,'sound/weapons/alien_tail_attack.ogg', 50, TRUE)
-	if(hit_target.mob_size < MOB_SIZE_BIG)
-		step_away(hit_target, xeno)
-
+	if(!step_away(hit_target, xeno))
```

Comment:
As mentioned on discord try instead `step` here likely in `stab_direction`. If `step` returns false entering this code you'll likely want that to induce `hit_target.apply_effect(0.5, WEAKEN)` if `if (hit_target.mob_size < MOB_SIZE_BIG)`

Reference https://github.com/cmss13-devs/cmss13/blob/5197640b085c59240d1449588371d3cc7bb3b01f/code/datums/ammo/ammo.dm#L138 for what all occurs for knockback/slamming.